perf/istio-install/setup_istio.sh

```diff
@@ -21,7 +21,7 @@ function download() {
   local outfile="${DIRNAME}/istio-${release}.tgz"
 
   if [[ ! -f "${outfile}" ]]; then
-    wget -O "${outfile}" "${url}"
+    curl -JLo "${outfile}" "${url}"
```

Comment:
wget on Alpine Linux does not follow redirects, while curl is more consistent in its behavior.